plenary/plenary-11.yaml

```diff
@@ -0,0 +1,235 @@
+metadata:
+  title: Resolutions adopted at the 11th meeting of ISO/TC 154
+  date: 1996-03-15
+  source: ISO/TC 154 Secretariat
+resolutions:
+  - dates:
+      - 1996-03-15
+    identifier: 100
+    
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: resolves
+        date_effective: 1996-03-15
+        message: |
+        ISO/TC 154 resolves to confirm the following ISO standards:
+        - ISO 6422
+        - ISO 8439
+        
+      - type: resolves
+        date_effective: 1996-03-15
+        message: |
+        ISO/TC 154 resolves to delete the following work items from the programme of work:
+        - ISO/CD 6423
+        - ISO/CD 6760
+        - ISO CD 10886
+        - ISO/NP 13183
+        - ISO/NP 13185
+        - ISO/NP 13186
+        - ISP/NP 13187
+
+  - dates:
+      - 1996-03-15
+    identifier: 101
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: requests
+        date_effective: 1996-03-15
+        message: ISO/TC 154 requests its secretariat to distribute the French member body contribution to the meeting of ISO/TC 154 (Doc. N 187) to all P members of ISO/TC 154 for comments. It is requested that comments be submitted via e-mail to the secretariat. Comments must be received by 1996-06-28. 
+
+        # An ad hoc group will be established to review the comments received and, if necessary, identify any changes to the present scope.
+        
+        # Members of the ad hoc group are:
+        
+        # Tuisko Buchmann
+        # Marcel Deturche
+        # Ingeborg Burger-Balogh
+        # Peter Wilson
+        # Billie Zidek
+        
+        # The results of the ad hoc group are expected to be submitted to the secretariat before the end of August 1996.
+        
+        # The results should provide the basis for the strategic policy statement of TC 154 which will be discussed at the next plenary meeting.
+        
+        
+  - dates:
+      - 1996-03-15
+    identifier: 102
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: advises
+        date_effective: 1996-03-15
+        message: ISO/TC 154 advises UN/ECE to submit the revised versions of ISO 9735 (V4) Parts 1, 2, 3 and 8 approved by UN/ECE as "existing standars" (according to ISO/IEC Directives Part 1 G.2.1.1) for ISO "Fast track procedure" within ISO
```

Comment:
Here, I used the word `advises` as type action. Is it right?

Reply:
Yes.